dotnet-efcore/RecordsAsModelClasses/RecordsAsModelClasses/Controllers/v2/CarsController.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using RecordsAsModelClasses.Core.Context;
+using RecordsAsModelClasses.Core.DTOs;
+using RecordsAsModelClasses.Core.Entities.Classes;
+
+namespace RecordsAsModelClasses.Controllers.v2
+{
+    [Route("api/v2/cars")]
+    [ApiController]
+    public class CarsController : ControllerBase
+    {
+        private readonly CarDbContext _context;
+
+        public CarsController(CarDbContext context)
+        {
+            _context = context;
+        }
+
+
+        [HttpPost]
+        public async Task<ActionResult<CarDto>> CreateCar([FromBody] CarDto carDto)
+        {
+            var car = new Car
+            {
+                Make = carDto.Make,
+                Model = carDto.Model,
+                Year = carDto.Year
+            };
+
+            _context.ClassCars.Add(car);
+            await _context.SaveChangesAsync();
+
+            return Created("api/v2/cars", car);
```

Comment:
Same here, this should be CreatedAtAction
```suggestion
            return CreatedAtAction(nameof(GetCar), new { id }, carDto);
```